services/QuillLMS/app/assets/stylesheets/pages/premium_hub/data-export.scss

```diff
@@ -1,6 +1,5 @@
 .data-export-main {
-  // this 317px comes from the filter sidebar element width
-  max-width: calc(100% - 317px) !important;
+  max-width: 1107px !important;
```

Comment:
Maybe add a note about where this number comes from in case we need to change it later?